<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomHeaderForm</class>
 <widget class="QWidget" name="RoomHeaderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>81</height>
   </rect>
  </property>
  <widget class="QWidget" name="roomNameContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="roomName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Clear Sans Medium</family>
     </font>
    </property>
    <property name="text">
     <string>Living Room</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="next">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="previous">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="mold::InvisibleClickableButton" name="nextBig">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="mold::InvisibleClickableButton" name="previousBig">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mold::InvisibleClickableButton</class>
   <extends>QPushButton</extends>
   <header>gui/InvisibleClickableButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
